src/crimson/os/seastore/transaction_manager.h

```diff
@@ -128,22 +128,24 @@ class TransactionManager : public ExtentCallbackInterface {
   template <typename T>
   pin_to_extent_ret<T> pin_to_extent(
     Transaction &t,
-    LBAPinRef pin) {
+    LBAMappingRef pin) {
     LOG_PREFIX(TransactionManager::pin_to_extent);
     SUBTRACET(seastore_tm, "getting extent {}", t, *pin);
     static_assert(is_logical_type(T::TYPE));
     using ret = pin_to_extent_ret<T>;
     auto &pref = *pin;
-    return cache->get_extent<T>(
+    return cache->get_extent<true, T>(
```

Comment:
Hmm, is it cache->get_extent<false, T>() ?

Reply:
yes, will correct it